modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -162,6 +163,17 @@ const CreationMenu = ({
 				>
 					<ClayIcon symbol="plus" />
 				</ClayLink>
+			) : (
+				<ClayLink
```

Comment:
Although anchor element without `href` is still [technically valid html](https://www.w3.org/TR/2011/WD-html5-20110525/links.html#attr-hyperlink-href), I don't think it's a good practice. We enforce it as a mandatory attribute in `<aui:a>`, but looks like not with `<ClayLink>`. I believe `button={true}` only adds CSS classes to make the the link look like a button.

Rather than using `<ClayLink>`, it would be cleaner to use `<ClayButtonWithIcon>`, like we are doing [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js#L40-L60). This also matches [similar cases in v2 implementation](https://github.com/liferay/clay/blob/2.x/packages/clay-management-toolbar/src/ClayManagementToolbar.soy#L282-L311).

Reply:
Sure Ok

Reply:
Changed [here](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js#L167-L174)